tests/ser.rs

```diff
@@ -25,8 +25,21 @@ function arrEqual(a, b) {
   return a.length === b.length && a.every((v, i) => v === b[i]);
 }
 "#;
+const JS_POLLUTE: &str = r#"
+Object.defineProperty(Array.prototype, "0", {
+  set: function (v) {
+    throw new Error("Should define own properties");
+  },
+});
+
+Object.defineProperty(Object.prototype, "a", {
+  set: (v) => {
+    throw new Error("WTF");
+  }
+});
+"#;
```

Comment:
The error messages of the two pollution cases should stay consistent, rather than one having an informative message (taken from https://github.com/tc39/test262/blob/main/test/built-ins/Array/of/does-not-use-prototype-properties.js) and one not (taken from my sample in #17).